classes/class3/Homework/public/catData.js

```diff
@@ -0,0 +1,17 @@
+var mongoose = require('mongoose');
```

Comment:
Keep server-side code out of the `/public/` directory - this is publicly accessible to anyone on the internet. A pretty big security flaw! You can put these schema files in a folder called `/model` in the future.